apps/docs/package.json

```diff
@@ -27,6 +27,7 @@
     "tailwind-merge": "^2.6.0",
     "unocss-preset-animations": "^1.2.1",
     "yaml": "^2.8.0",
+    "zod": "^3.25.67",
```

Comment:
Add to add this so `zod-to-json-schema` doesn't peer-depend on zod v4 from vitest